CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [11.1.1](https://github.com/theforeman/puppet-foreman_proxy_content/tree/11.1.1) (2020-08-27)
```

Comment:
Can you include a `Full Changelog` link like done for the others? Once the tag shows up, it should work.
```suggestion
## [11.1.1](https://github.com/theforeman/puppet-foreman_proxy_content/tree/11.1.1) (2020-08-27)

[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy_content/compare/11.1.0...11.1.1)
```